iothub/service/src/IotHubServiceClientOptions.cs

```diff
@@ -59,6 +61,18 @@ public IotHubServiceClientOptions()
         /// </summary>
         public HttpClient HttpClient { get; set; }
 
+        /// <summary>
+        /// Whether to use web sockets or not (Only used for AMQP or MQTT).
+        /// </summary>
```

Comment:
```suggestion
        /// Whether to use web sockets or not.
        /// </summary>
        /// <remarks>
        /// Only used for AMQP or MQTT.
        /// </remarks>
```

Reply:
There is no MQTT API in the service client